fortitude/src/message/rdjson.rs

```diff
@@ -0,0 +1,138 @@
+use std::io::Write;
+
+use serde::ser::SerializeSeq;
+use serde::{Serialize, Serializer};
+use serde_json::{json, Value};
+
+use ruff_diagnostics::Edit;
+use ruff_source_file::SourceCode;
+use ruff_text_size::Ranged;
+
+use crate::message::{Emitter, SourceLocation};
+
+use super::DiagnosticMessage;
+
+#[derive(Default)]
+pub struct RdjsonEmitter;
+
+impl Emitter for RdjsonEmitter {
+    fn emit(
+        &mut self,
+        writer: &mut dyn Write,
+        messages: &[DiagnosticMessage],
+    ) -> anyhow::Result<()> {
+        serde_json::to_writer_pretty(
+            writer,
+            &json!({
+                "source": {
+                    "name": "ruff",
+                    "url": "https://docs.astral.sh/ruff",
```

Comment:
Left some ruff bits here
```suggestion
                    "name": "fortitude",
                    "url": "https://github.com/PlasmaFAIR/fortitude",
```